EduPerfTests/Program.cs

```diff
@@ -229,20 +303,20 @@ static RemoteWebDriver LaunchDriver(Browser browser)
                         Thread.Sleep(2000);
                         break;
                     default:
-                        throw new Exception(string.Format("Unexpected browser: {0}", browser));
+                        throw new Exception($"Unexpected browser: {browser}");
                 }
 
-                driver.Manage().Window.Maximize();
+                //driver.Manage().Window.Maximize();
```

Comment:
You may want to reconsider this. Not all browsers launch maximized (Chrome mainly but others as well). This could impact memory tests because you could get a mobile/tablet version of a site with less payload (helps to guarantee desktop version if it's maximized although even that isn't foolproof).